packages/serverless-nextjs-component/README.md

```diff
@@ -93,6 +93,27 @@ And simply deploy:
 $ serverless
 ```
 
+### Custom Cloudfront configuration
```

Comment:
Can we link this section in the top of the docs (Contents)

Reply:
Sure thing...